ms-engine/src/main/java/net/trajano/ms/engine/SpringJaxRsHandler.java

```diff
@@ -257,9 +260,9 @@ private Client jaxRsClient(final Vertx vertx) {
 
         Client jaxRsClient = vertx.getOrCreateContext().get(Client.class.getName());
         if (jaxRsClient == null) {
-            //            final HttpClient httpClient = vertx.createHttpClient(httpClientOptions);
-            //            jaxRsClient = new ResteasyClientBuilder().providerFactory(providerFactory).httpEngine(new VertxClientEngine(httpClient)).build();
-            jaxRsClient = new ResteasyClientBuilder().providerFactory(providerFactory).httpEngine(new ApacheHttpClient43Engine()).build();
+            final HttpClient httpClient = vertx.createHttpClient(httpClientOptions);
+            jaxRsClient = new ResteasyClientBuilder().providerFactory(providerFactory).httpEngine(new VertxClientEngine(httpClient)).build();
+            //            jaxRsClient = new ResteasyClientBuilder().providerFactory(providerFactory).httpEngine(new ApacheHttpClient43Engine()).build();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)